### PR TITLE
Add getting started instructions to readme

### PR DESCRIPTION
**Pull request by connor11528**

### CODE REVIEW

**Comment by connor11528** [comment]
This works to get it up and running. Documentation is helpful

File: README.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+> A demonstration boilerplate for generating a static with with React and webpack
+
+### Getting Started
+
+```
+$ git clone git@github.com:jxnblk/react-static-site-boilerplate.git
+$ cd react-static-site-boilerplate
+$ npm install
+$ npm start
+$ npm i -g http-server
```
Comment (jxnblk) [inline README.md]: What are you using http-server for here?

**Comment by connor11528** [comment]
My bad. http-server not necessary. `npm start` starts the server

**Comment by ivanoats** [comment]
LGTM! :shipit: please merge this.

**Comment by jxnblk** [comment]
Thanks! Should probably add `rm -rf .git` to the instructions. Also, is that a blockquotes for the first paragraph?

**Comment by connor11528** [comment]
I mean removing git isn't necessary to make it run. I can make changes if necessary. Yeah it's a blockquote, I like the way it looks for a description

**Comment by jxnblk** [comment]
I don't think a blockquote is semantically correct in this case.

**Pull request closed without merging**